recipes/cis_interface/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "cis_interface" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 441e6b2a3b409b5e8373988b4d4d0c88920af8cecc3b599fa89f3191f07d638d
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win and vc<14]
+  entry_points:
+    - cisrun = cis_interface.command_line:cisrun
+    - ciscc = cis_interface.command_line:ciscc
+    - cisccflags = cis_interface.command_line:cc_flags
+    - cisldflags = cis_interface.command_line:ld_flags
+    - cistest = cis_interface:run_nose
+    - cisschema = cis_interface.command_line:regen_schema
+    - cisconfig = cis_interface.command_line:update_config
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - czmq
+    - pip
+    - python
+    - sysv_ipc  # [unix]
+    - zeromq
+  run:
+    - cerberus
+    - czmq
+    - jsonschema
+    - matplotlib
+    - nose
+    - numpy
+    - pandas
+    - pint
+    - psutil
+    - pystache
+    - python
+    - pyyaml
+    - pyzmq
+    - scipy
+    - sysv_ipc  # [unix]
+    - unyt
+    - zeromq
+
+test:
+  imports:
+    - cis_interface
+  requires:
+    - {{ compiler('c') }}
```

Comment:
Just to double check here, do you need the compilers for test or just the libraries?

Reply:
huuumm.. ok, I just saw you last commit  :)